Entity/UserManager.php

```diff
@@ -146,4 +148,28 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = arra
 
         return $pager;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loadUserByUsername($username)
+    {
+        return $this->findUserByUsername($username);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function refreshUser(SecurityUserInterface $user)
+    {
+        return $this->loadUserByUsername($user->getUsername());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsClass($class)
```

Comment:
`public function supportsClass($class): bool`

Reply:
:D Ok, great, should I do the rest of the class as well? Or just the new methods?

Reply:
Just the new methods, otherwise it would be a BC-break